packages/synapse-react-client/src/lib/style/components/entity_finder/_entity-finder.scss

```diff
@@ -36,69 +36,17 @@ $-splitter-width: 12px;
   width: 100%;
   max-width: max-content;
   margin: auto;
-  &__Search {
-    display: flex;
-    justify-content: flex-end;
-    align-items: center;
-    margin: 10px 0px 10px 0px;
-    height: 38px;
-    .SearchIcon {
-      position: relative;
-      left: 27px;
-      top: 1px;
-    }
-    .ClearSearchIcon {
-      color: map.get(SRC.$primary-color-palette, 500);
-      cursor: pointer;
-      position: relative;
-      left: -30px;
-      margin-right: -8px;
-    }
-    button.btn {
-      height: 100%;
-    }
-    button.btn:hover {
-      svg {
-        path {
-          fill: white;
-        }
-      }
-    }
-    &__SearchButton {
-      svg {
-        margin-right: 10px;
-        margin-bottom: 2px;
-      }
-    }
 
-    input.EntityFinder__Search__Input {
-      border: 1px solid SRC.$gray-dark;
-      background-color: transparent;
-      height: 100%;
-      margin: 10px 0px;
-      padding: 5px 28px;
-      box-shadow: none;
-      transition: flex-grow 250ms ease-in-out, width 250ms ease-in-out;
-    }
-  }
-
-  &__Search[data-active='true'] {
-    input.EntityFinder__Search__Input {
-      width: 100%;
+  &__Search__Input {
+    transition: flex-grow 250ms ease-in-out, width 250ms ease-in-out;
```

Comment:
Bring back the search field transition -- 

https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/3b263fff-6e7d-4419-8412-24333230edf7